src/core/janeway_global_settings.py

```diff
@@ -308,7 +308,7 @@ def gettext(s):
 BLEACH_STRIP_TAGS = True
 
 # Strip comments, or leave them in.
-BLEACH_STRIP_COMMENTS = False
+BLEACH_STRIP_COMMENTS = True
```

Comment:
can we add a comment here pointing at #4695 to alert others who may want to override this setting in their own environment